README.md

```diff
@@ -147,7 +147,7 @@ Supported options:
 | :---- | :------: | :--- |
 | url | string   | URL you want to share (you can share a base64 file url only in iOS & Android ) |
 | urls | Array[string]   | URL's you want to share, Only for iOS and Android (you can share a base64 file url only in iOS & Android ) |
-| type | string   | File mime type (optional) |
+| type | string   | File mime type (optional).The support types are as follows, and must be shared when sharing files. |
```

Comment:
| type | string   | File mime type (optional). The support types are as follows, and must be used when sharing files. |